vertical-pod-autoscaler.yaml

```
@@ -0,0 +1,65 @@
+package:
+  name: vertical-pod-autoscaler
+  version: 1.27.1
+  epoch: 0
+  description: Vertical Pod Autoscaler (VPA) frees users from the necessity of setting up-to-date resource limits and requests for the containers in their pods.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/autoscaler
+      tag: v${{package.version}}
+      expected-commit: bd7a995e68b8d79d1c7fbc55946f3ca3c441099e
+      destination: autoscaler
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin
+      cd autoscaler/vertical-pod-autoscaler/pkg
+      cd ./admission-controller
+      make build-binary
+      install -Dm755 ./admission-controller "${{targets.destdir}}/usr/bin/admission-controller"
+      cd ../recommender
+      make build-binary
+      install -Dm755 ./recommender "${{targets.destdir}}/usr/bin/recommender"
+      cd ../updater
+      make build-binary
+      install -Dm755 ./updater "${{targets.destdir}}/usr/bin/updater"
+      cd ../../..
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-admission-controller
+    pipeline:
+      - runs: |
+          install -Dm755 ${{targets.destdir}}/usr/bin/admission-controller ${{targets.subpkgdir}}/usr/bin/admission-controller
+
+  - name: ${{package.name}}-recommender
+    pipeline:
+      - runs: |
+          install -Dm755 ${{targets.destdir}}/usr/bin/recommender ${{targets.subpkgdir}}/usr/bin/recommender
+
+  - name: ${{package.name}}-updater
+    pipeline:
+      - runs: |
+          install -Dm755 ${{targets.destdir}}/usr/bin/updater ${{targets.subpkgdir}}/usr/bin/updater
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes/autoscaler
+    strip-prefix: v
```

looks like tags are in the format `addon-resizer-<version>`
```suggestion
    strip-prefix: cluster-autoscaler-
```